docs/filters.md

```diff
@@ -321,3 +321,27 @@ HubTurbo's filter system is incompatible with GitHub's in a number of ways.
     + `closed:[date range]`
     + `mentions:[string]`
     + `commenter:[string]`
+
+## Cheatsheet
+|Keywords|Aliases|Params aliases|
```

Comment:
Maybe the first column could be `Qualifiers`, and the third could be `Keywords`. We overload 'keywords' quite a lot but that's the most general term for these.